src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -282,64 +281,7 @@ protected override void ComputeTrainingStatistics(IChannel ch, FloatLabelCursor.
                 }
             }
 
-            // Apply Cholesky Decomposition to find the inverse of the Hessian.
```

Comment:
>Apply Cholesky Decomposition to find the inverse of the Hessian. [](start = 15, length = 64)

Hmm, it's too bad we have to completely drop good code.
Is it possible we could somehow move it to the MKL-dependent assembly? At least have a REVIEW for this.

Reply:
Maybe have a virtual 'UpdateStdErrors' method that does nothing here, and have a better version of LR in MKL-0dependent assembly that overrides this?

---
In reply to: [206635936](https://github.com/dotnet/machinelearning/pull/611#discussion_r206635936) [](ancestors = 206635936)

Reply:
we're not dropping it. logged [https://github.com/dotnet/machinelearning/issues/612](https://github.com/dotnet/machinelearning/issues/612) to do exactly that, move the code to the mkl dependent package. 


---
In reply to: [206635936](https://github.com/dotnet/machinelearning/pull/611#discussion_r206635936) [](ancestors = 206635936)

Reply:
Got it

---
In reply to: [206637501](https://github.com/dotnet/machinelearning/pull/611#discussion_r206637501) [](ancestors = 206637501,206635936)